package/log/log.go

```diff
@@ -42,7 +42,7 @@ type Flusher interface {
 	Flush()
 }
 
-type Logger interface {
+type Interface interface {
```

Comment:
log.Logger would always default to the standard libary's log package. This fixes that.